Package.swift

```diff
@@ -4,7 +4,7 @@ import PackageDescription
 let package = Package(
     name: "Crypto",
     products: [
-        .library(name: "Crypto", targets: ["Crypto"]),
+        .library(name: "Crypto", targets: ["Crypto", "libbcrypt"]),
```

Comment:
Same comment as other pr, we should keep `libbcrypt` internal (an implementation detail).